File: jogo-app-back/JogoApi.DTO/RetornoRodada.cs
using Newtonsoft.Json;
using System.Collections.Generic;

namespace JogoApi.DTO
{
    [JsonObject(Title = "NovaRodada")]
    public class RetornoRodada
    {
        [JsonProperty(PropertyName = "CodigoJogo")]
        public int CodigoJogo { get; set; }

        [JsonProperty(PropertyName = "NumeroRodada")]
        public int NumeroRodada { get; set; }

        [JsonProperty(PropertyName = "Diretorio")]
        public string Diretorio { get; set; }

        [JsonProperty(PropertyName = "Palavras")]
        public List<JogoPalavras> Palavras { get; set; }
    }
}

File: jogo-app-front/src/app/models/RetornoRodada.ts
export interface RetornoRodada {
     CodigoJogo: number,
     Diretorio: string,
     NumeroRodada: number,
     Palavras: [
        {CodigoAcerto: number, Palavra: string},
        {CodigoAcerto: number, Palavra: string},
        {CodigoAcerto: number, Palavra: string},
        {CodigoAcerto: number, Palavra: string}
]
}